src/main/scala/uk/gov/homeoffice/drt/service/ArrivalService.scala

```diff
@@ -0,0 +1,33 @@
+package uk.gov.homeoffice.drt.service
+
+import cats.effect.Sync
+import cats.syntax.all._
+import org.slf4j.LoggerFactory
+import uk.gov.homeoffice.drt.{ResourceObject, model}
+import uk.gov.homeoffice.drt.model.{Arrival, DepartureAirport, RequestedFlightDetails}
+import uk.gov.homeoffice.drt.repository.{ArrivalRepositoryI, ArrivalTableData}
+import uk.gov.homeoffice.drt.utils.DateUtil._
+import ResourceObject.{getClass, _}
+class ArrivalService[F[_] : Sync](arrivalsRepository: ArrivalRepositoryI[F]) {
+
+  private val logger = LoggerFactory.getLogger(getClass.getName)
+
+  def getFlightsDetail(requestedDetails: RequestedFlightDetails) = {
+    val requestedDate = parseLocalDate(requestedDetails.date).atStartOfDay()
+    val athensCountryAirport = DepartureAirport.getAthensCountryAirport(requestedDetails.country)
+    arrivalsRepository.findArrivalsForADate(requestedDate).map(_.filter(athensCountryAirport.map(_.airportCode) contains _.origin).zipWithIndex)
+  }
+
+
+  def transformArrivals(arrivalsTableData: F[List[(ArrivalTableData, Int)]]): F[List[Arrival]] = {
+    arrivalsTableData.map(_.map(a => model.Arrival((a._2 + 1).toString, dateDefaultTimeZoneConvert(a._1.scheduled), carrierName(a._1.code,a._1.number.toString), a._1.code.toString, a._1.destination, a._1.origin, a._1.scheduled_departure.map(dateAthensTimeZoneConvert(_)))))
```

Comment:
This would be easier to read if you use `case`s to name the tuple parts. Otherwise I don't know what `a._1` or `a._2` is without opening up intellij and inspecting it